MQuince/MQuince.Integration.HospitalApp.Tests/PharmacyMapperTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using MQuince.Integration.Entities;
+using MQuince.Integration.Repository.MySQL.DataProvider.Util;
+using MQuince.Integration.Repository.MySQL.PersistenceEntities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MQuince.HospitalAppTests
+{
+    public class PharmacyMapperTest
+    {
+        [Fact]
+        public void MapPharmacyPersistenceToPharmacyEntity_Given_null_should_be_null()
+        {
+            MyPharmacy pharmacy = PharmacyMapper.MapPharmacyPersistenceToPharmacyEntity(null);
+
+            Assert.Null(pharmacy);
+        }
+        [Fact]
+        public void MapPharmacyPersistenceToPharmacyEntity_Given_some_value_should_back_new_MyPharamcy()
+        {
+
+            var pharmacyTest = new PharmacyPersistence(new Guid(), "ApotekaTest", "apoteka.com");
+
+            MyPharmacy pharmacy = PharmacyMapper.MapPharmacyPersistenceToPharmacyEntity(pharmacyTest);
+
+            Assert.NotNull(pharmacy);
```

Comment:
Assert should check if each property is mapped correctly, to assert that return value of method is not null is not enough